### DISCUSSION TITLE
2023년 07월 06일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#66] [General]
# 2023년 07월 06일

LOG 점심으로 라면을 먹었다. 싸다김밥의 싸다는 cheap이 아니라 wrap이었다고 한다. by 바론 선호, 수민 언니, 세진과 뚝섬에서 파스타를 먹었다. 사실 그렇게 맛있진 않았다..ㅋㅋㅋ 토마토 파스타가 맛이 없기 쉽지 않은데 배운 것 repo…

https://vero.wiki/2023-07-06/

**Comment by carsago** [comment]
베로 잠실에서 외롭다는 소문이 있던데

**Reply by Cyma-s** [reply]
오리 잠실 놀러와 ㅠ
